packages/lit-dev-content/src/global/hydrate-common-components.ts

```diff
@@ -29,3 +31,63 @@ for (const search of searches) {
     {once: true}
   );
 }
+
+// When in mobile and in the docs section, opens the <details> element in the
+// mobile nav drawer corresponding to the current docs page. Additionally, if
+//that section is out of view because the navigation list is too long, scrolls
+// it into view.
+const activateMobileDetails = async () => {
+  // Expand the current section in the mobile nav.
+  const activeDetails = document.querySelector(
+    '#mobileDocsNav > li.activeSection details'
+  ) as HTMLDetailsElement | null;
+  if (activeDetails === null) {
+    return;
+  }
+  activeDetails.open = true;
+
+  const drawer = document.body.querySelector('mwc-drawer');
+  // Wait for the drawer to begin opening.
+  drawer?.addEventListener(
+    'opening',
```

Comment:
Why is this additional `opening` event required?

Reply:
TBH I'm pretty sure it never worked before...

confirmed it never worked before:

https://lit.dev/docs/resources/community/
https://pr851-041c2a2---lit-dev-5ftespv5na-uc.a.run.app/docs/resources/community/

open those pages on mobile, make the height very low, and open the side drawer. The `Community` thing should be visible on both

Reply:
https://user-images.githubusercontent.com/5981958/172723304-8b05337d-08b1-4398-9a0c-4fa1489b1fd2.mov

Reply:
Great repro! Great fix!